cetmix_tower_server/views/cx_tower_server_template_view.xml

```diff
@@ -176,12 +176,20 @@
                                 <tree editable="bottom">
                                     <field name="variable_reference" invisible="1" />
                                     <field name="variable_id" />
-                                    <field name="value_char" />
+                                    <field
+                                        name="value_char"
+                                        attrs="{'required': [('required', '=', True)]}"
+                                    />
+                                    <field name="required" />
                                 </tree>
                                 <form>
                                     <group>
                                         <field name="variable_id" />
-                                        <field name="value_char" />
+                                        <field
+                                            name="value_char"
+                                            attrs="{'required': [('required', '=', True)]}"
```

Comment:
According to the task: `For all checked lines this configuration variable must be a mandatory field in the New Server Wizard:`
This is not a Server Wizard.